knotx-stack-manager/src/main/packaging/conf/includes/serviceAdapter.conf

```diff
@@ -17,6 +17,10 @@ clientOptions {
   #
   # Hostname verification
   # verifyHost = false
+  #
+  # Enables network activity logger (Netty). Remember you also need to include additional logger
+  # configuration in the logback config.
```

Comment:
Maybe tell here which logger should be added in logback.xml, simply an xml snipept of that logger